server/etcdserver/server.go

```diff
@@ -2067,6 +2081,8 @@ func (s *EtcdServer) applyEntryNormal(e *raftpb.Entry) {
 	// raft state machine may generate noop entry when leader confirmation.
 	// skip it in advance to avoid some potential bug in the future
 	if len(e.Data) == 0 {
+		s.notifyAboutFirstCommitInTerm()
+
 		select {
 		case s.forceVersionC <- struct{}{}:
```

Comment:
Please add at least a TODO to replace forceVersionC with FirstCommitInTermNotify as they do exactly the same thing.